references/sdk.md

```diff
@@ -2,12 +2,35 @@
 
 # Client
 ```python
-Client(self, url: Union[str, NoneType] = None, token: Union[str, NoneType] = None, retry_timeout: int = 300, backend: str = 'remote', version: str = '0.0', insecure: bool = False)
+Client(self, url: Union[str, NoneType] = None, token: Union[str, NoneType] = None, retry_timeout: int = 300, version: str = '0.0', insecure: bool = False, debug: bool = False)
 ```
-Create the client.
+Create a client
 
-Set the `backend` to 'local' to use the local debugging.
-One client corresponds to one profile.
+Args:
+
+    url (str, optional): URL of the Substra platform. Mandatory
+        to connect to a Substra platform. If no URL is given and debug is True, all
+        assets must be created locally.
+        Defaults to None.
+
+    token (str, optional): Token to authenticate to the Substra platform.
+        If no token is given, use the 'login' function to authenticate.
+        Defaults to None.
+
+    retry_timeout (int, optional): Number of seconds before attempting a retry call in case
+        of timeout.
+        Defaults to 5 minutes.
+
+    version (str, optional): API version, do not change it. Defaults to '0.0'.
+
+    insecure (bool, optional): If True, the client can call a not-certifed backend. This is
+        for debugging purposes.
```

Comment:
development purposes? I would avoid the word debugging to not confuse with the next `debug` option.